### DISCUSSION TITLE
Horovod vs Databrick's Horovod Runner

### DISCUSSION THREAD

**Discussion by toncho11** [horovod/horovod#3749] [General]
So Horovod is designed to work on a single machine with multiple GPUs while Databrick's HorovodRunner allows you run Horovod as Spark jobs effectively allowing multiple machines(nodes) to be used for training. Is that correct?

**Comment by maxhgerlach** [comment]
I don't use Horovod with Spark, so I can't comment on that or Databricks.

But you can very much use Horovod for large distributed trainings on multiple nodes with multiple GPUs each. See the docs for pointers to multiple options of running Horovod (personally, I mostly use MPI): https://horovod.readthedocs.io/en/stable/summary_include.html#running-horovod
